### DISCUSSION TITLE
Compatible MKL versions

### DISCUSSION THREAD

**Discussion by chillenb** [block-hczhai/block2-preview#92] [Q&A] [answered]
Hi! I have a question about which MKL versions are compatible with block2.
In setup.py, the MKL version is set as mkl==2021.4. What happens if you try to use a newer version?

**Comment by hczhai** [comment, marked as answer]
The restriction of MKL version is only applied to ``pip install block2``. If you manually compile ``block2`` using ``cmake`` following the instructions on the page https://block2.readthedocs.io/en/latest/user/installation.html#manual-installation, you can use any MKL version or even without MKL.

**Reply by chillenb** [reply]
Thanks for the clarification! I will try it out.
